package.json

```diff
@@ -29,10 +29,8 @@
   "dependencies": {
     "@readme/data-urls": "^1.0.1",
     "@types/har-format": "^1.2.12",
-    "readable-stream": "^3.6.0"
-  },
-  "optionalDependencies": {
-    "formdata-node": "^4.3.2"
+    "readable-stream": "^3.6.0",
```

Comment:
there's no way to conditionally load this right?

Reply:
Could do an `await import()` but I think even if this source gets compiled by Webpack it would still have to load it in the compiled source.